README.md

```diff
@@ -161,7 +161,7 @@ I recommend using Google, GitHub Issues, or Uptime Kuma's Subreddit for finding
 - [GitHub Issues](https://github.com/louislam/uptime-kuma/issues)
 - [Subreddit r/Uptime kuma](https://www.reddit.com/r/UptimeKuma/)
 
-My Reddit account: [u/louislamlam](https://reddit.com/u/louislamlam).  
+My Reddit account: [u/louislamlam](https://reddit.com/u/louislamlam).
```

Comment:
The other whitespaces don't matter, but this one does change the layout from two small to one line. (double whitespace at the end of lines is treated as enter in markdown)
```suggestion
My Reddit account: [u/louislamlam](https://reddit.com/u/louislamlam).  
```